packages/web-twig/src/Resources/components/CheckboxField/stories/CheckboxField.twig

```diff
@@ -0,0 +1,11 @@
+<CheckboxField
+    id="checkboxfield1"
+    label="Checkbox unselected"
+    name="checkboxfield1"
+/>
+<CheckboxField
+    id="checkboxfield5"
+    isChecked={true}
```

Comment:
It could be just isChecked, right? We don't need to specify the value if it is boolean and we set it true

Reply:
same for all other boolean props in this PR

Reply:
Done